pkg/parser/task/command/cache.go

```diff
@@ -39,20 +41,25 @@ func NewCacheCommand(mergedEnv map[string]string, boolevator *boolevator.Booleva
 	})
 
 	folderSchema := schema.String("Path of a folder to cache.")
-	cache.RequiredField(nameable.NewSimpleNameable("folder"), folderSchema, func(node *node.Node) error {
+	cache.OptionalField(nameable.NewSimpleNameable("folder"), folderSchema, func(node *node.Node) error {
 		folder, err := node.GetExpandedStringValue(mergedEnv)
 		if err != nil {
 			return err
 		}
 
-		// https://github.com/cirruslabs/cirrus-ci-agent/issues/47
-		const homePrefix = "~/"
-		folder = strings.TrimSpace(folder)
-		if strings.HasPrefix(folder, homePrefix) {
-			folder = "$HOME/" + strings.TrimPrefix(folder, homePrefix)
+		cache.folder = folder
```

Comment:
Let's remove `folder` and use only `folders like that:

```suggestion
		cache.folders = []string{folder}
```